client/src/procedures/primitives.rs

```diff
@@ -347,17 +350,56 @@ impl FromStr for MnemonicLanguage {
     }
 }
 
+pub mod serde_bip39_mnemonic {
+    use super::bip39;
+
+    pub fn serialize<S>(m: &bip39::Mnemonic, s: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        s.serialize_str(m.as_ref())
+    }
+
+    pub fn deserialize<'de, D>(d: D) -> Result<bip39::Mnemonic, D::Error>
+    where
+        D: serde::Deserializer<'de>,
+    {
+        use serde::Deserialize;
+        String::deserialize(d).map(bip39::Mnemonic::from)
+    }
+}
+
+pub mod serde_bip39_passphrase {
```

Comment:
You should be able to use the same for both, something like that https://github.com/iotaledger/iota-sdk/blob/develop/sdk/src/utils/serde.rs#L4C10-L4C10

Reply:
Thanks, didn't know that. Fixed.